microservices/clients/members-client/src/main/java/org/xcolab/client/members/MembersClient.java

```diff
@@ -330,9 +330,13 @@ public static boolean updateMember(Member member) {
     }
 
     public static boolean deleteMember(long memberId) {
-        return memberResource.delete(memberId)
+        boolean result = memberResource.delete(memberId)
                 .cacheName(CacheName.MEMBER)
                 .execute();
+        if (result) {
+            ServiceRequestUtils.clearCache(CacheName.MEMBER_LIST);
```

Comment:
Was this the fix to the caching problem or is there still a delay in the deletion?